### PR TITLE
Task/html template works

### PR DESCRIPTION
**Pull request by JuanGro**
#### What's this PR do?
To verify the works template.

#### Where should the reviewer start?
With all the fields.

#### How should this be manually tested?
It doesn't apply.

#### Any background context you want to provide?
No.

This template was adapted ~stolen~ from [Quickleft/Sprint.ly](https://quickleft.com/blog/pull-request-templates-make-code-review-easier/)

### CODE REVIEW

File: src/app/general-user/works/work-form.component.ts [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+* - Request save the work.
+* - Request actions in modals to the parent component.
+**/
+export class WorkFormComponent implements OnChanges {
+  // Requests close of the current modal to parent component.
+  @Output() requestCloseModal: EventEmitter<string> = new EventEmitter();
+  // Requests to parent component the show of the danger modal to confirm if the contact is permanent removed.
+  @Output() requestWarning: EventEmitter<string> = new EventEmitter();
+  // Variable to check if the submitForm method finish correctly.
+  public success: boolean = false;
```
Comment (EduardoVaca) [inline src/app/general-user/works/work-form.component.ts]: Delete this variable. We don't need it

File: src/app/general-user/works/work-form.component.html [hunk 2]
```diff
@@ -0,0 +1,252 @@
...
+        </div>
+    </div>
+    
+    <!-- Third row -->
+    <div class="form-group row">
+        <div class="col-md-6">
+            <!-- Type field -->
+            <label for="recipient-type" class="form-control-label">Tipo:</label>
+            <div class="input-group">
+                <span class="input-group-addon"><i class="fa fa-suitcase"></i>
```
Comment (EduardoVaca) [inline src/app/general-user/works/work-form.component.html]: Please choose icons more related to the field

File: src/app/general-user/works/work-form.component.ts (hunk 1 above)
Comment (JuanGro) [inline src/app/general-user/works/work-form.component.ts]: Done

File: src/app/general-user/works/work-form.component.html (hunk 2 above)
Comment (JuanGro) [inline src/app/general-user/works/work-form.component.html]: Done

**Pull request merged**